### PR TITLE
Arakkoa Faiths and Names

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Added 3 arakkoa faiths in their own group, Avianism:
  - Anzuism
  - Rukhmarism
  - Sethekk
- Added name sets for the Arakkoa:
<details>
<summary>Male</summary>

```
Akizzi Anjuk Bra'tkireerr Brakma Bralksaryiks Bralkseth Braylthrynis Brith Brokk Cha'kym Chaks
Chal Chazrir Che'kiss Chigikks Chikron Cho'kelyaik Cho'kkaa Chod Cholnutiz Chorr Desh Dezyr
Do'skerr Dokmi Duah Duatkyx E'skox Ekkizal Ekktyrr Ekroas Elksuruuas Engeylyos Er'thevok Esrysh
Essku Evrorruath Fad Falnukkox Fazee Fekrylyekk Fesfekkeeth Fish Fuuagaak Graasfyss Graassku
Gresh Gresskotix Grezox Gribiz Grikluk Grish Gruatkykk Gruuahaz I'rroh I'saves I'zo Ikzux
Ilazzar Isfetakks Issakks Issuvesh Ivraarrif Izzezikk Jaks Jaynyryoa Jec Jekks Jiktyves Jisskar
Jongoozearr Jot Kakks Kakra Kektorras Kerykks Kik Kizzyrra Kokrozzex Koktuta Kosfor Krasskonef
Kre'skeyroax Krikks Krivrevir Krohoc Krosoz Krostesh Krowk Kuuakzyf Laikk Lalkorr Lekmeryos
Lerran Libeetess Litkinix Lokks Lokleros Makk Malketaks Marath Marshi Mebivir Meth Mex Meze Mok
Morduf Mos O'tooryac Oklitir Okres Olurr Ongyvaf Onjuss Ordis Ordok Orruneth Pa'tanir Pam
Pe'syth Pershynaath Pess Pih Pokk Por Pordu Ranjevikks Rardirress Rawk Re'rrykke Reeth Rehis
Reth Ribykks Ril Rolnaryak Ross Sakk Sayk Sazzuss Selksyrr Sikutoth Skak Skarsheetakks Skikks
Skokks Skovrum Skrakk Skrazrokketh Skrekktyr Skrelksiks Skrerdex Skrorr Sra'skesh Sragas
Sralkezoks Srash Sratkez Sre'rynaad Sreshkuur Sri'sku Srikkiss Srir Srix Srolkush Srotketim
Sruas Ssatkok Ssedorr Sseeth Sselksuux Ssessukkuuaks Ssir Sso'skukk Ssonjulyiks Ssosh Tah Te'ruu
Tektara Teth Tetkulyoax Thayrri Thi'zurikks Thik Thothytin Tidoavekk Tihym Tirdass Varshok
Velthreerr Vesh Vok Vozas Zaashnes Zah Zalkur Zar Zar'thozzakk Zasfekk Zavrysh Zinjezzash Zith
Zo'keryor Zo'skash Zoks Zollik Zoth Zuad
```

</details>

<details>
<summary>Female</summary>

```
A'kkex Adax Aekhex Aekhissex Alrax Alsrax Alsresex Alzres Arakat Avax Ekith Ekligaf Eklirri
Ektisi Elissy Elksec Eogiky Eolksax Eolsex Erzessax Faklex Falrirrath Falzassyx Felis Firze
Fokix Fra'kkix Frae'kkassat Frelthyx Frokkax Froklerrif Froltex Frulzyx Fugy I'kix Ikix Ilsrit
Iltix Irkyx Irnix Itharrex Ithyx Jalen Jelri Jorasex Jukka Julsezex Ka'rres Kae'kax Kilsrax
Kraelzax Kraheta Kralksex Kregix Kreokhix Kroklex Krorkix Krovim Kru'kix Kulthezzix Kurlax
Kuvane Lalsrekax Lashe Lelzra Lilksex Lirkrez Lo'rrisa Lornix Lulsezzath Madekkix Mae'kkit
Maretyx Mekhati Meltyt Merken Mo'kyta Modakyx Molthex Mukkix Mulzrirrex Murlix Nadave Naeltanym
Nakka Negex Neoklex Nesheze Ni'kanix Ni'kyx Ni'rrex Niktin Nilzrix Nolkset Nu'kix Nulksi
O'karrec O'rarrix Oda Okhax Okhyt Okla Oklyx Olksinix Orax Orkax Orla Orzizam Othys Rarlynix
Rernyx Rilsax Ro'ka Rolra Rulrix Sagyth Salksix Salthix Selsran Shaeren Shaktix She'kkazzeth
Shekhava Shelax Shelsex Shelthakaf Shelzem Shihax Shishetex Sholsam Shralthyx Shrarlissax
Shrirla Shruklix Shrushex Shuhessa Shuktyzzyx Silksyx Ska'rrarrax Skarzi Skekam Skeorec Skerlex
Skihem Skirkraf Skivex Skukex Skulzrex Solzex Sorlac Sornyzi Sra'kkex Sraklyx Srarkrasa Sre'kax
Srilriryh Srilrix Srishex Srogyx Srolksaga Srulksax Sseo'rrizzen Sseran Ssi'ra Ssigix Ssirkax
Ssirkry Ssohym Ssorkres Ssovekkyx Ssulksit Ssurlirri Su'rex Su'rrix Sukhis Sulzryx Tharzat
Thi'kix Thilryssa Thogex Thohax Thokhyx Thoktysyx Tholsex Thu'ketex Thu'ryx Thulzan Ukex
Ukkarrex Ukkix Uktaf Ulrax Ulthath Ultyx Vahassex Valzregat Ve'rrax Vekkix Vikka Vo'rrex
Vurzerrix Za'kki Zahyx Zalsryth Zalzekkax Ze'rerrax Zeornasyx Zi'rresin Zihyx Zilzith
```

</details>

# How to test:
@Warcraft-GoA-Development-Team/testers, open the **_Religion Tab_**,  click **_Other Faiths_**, check **_Only Your Faiths_**, find these new faiths.
Tell your opinion about tenets, doctrines, descriptions.

@Warcraft-GoA-Development-Team/english-proofreaders, [Filtered Only Loc Files](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/401/files?file-filters%5B%5D=.yml).
Feel free to change the texts as you want, but keep the same length.

### CODE REVIEW

File: localization/english/religion/wc_religion_l_english.yml [hunk 1]
```diff
@@ -660,3 +660,27 @@
...
+ avian_group_adj:0 "Avian"
+ avian_group_adherent:0 "Avian"
+ avian_group_adherent_plural:0 "Avians"
+ avian_group_desc:0 "A group of faiths aroused among the arakkoa of Draenor consisting of four $avian_group_adherent$ gods: Rukhmar, Anzu, Sethe and the greatest king Terokk they worship as their fourth god."
+ 
+ anzuism:0 "Anzuism"
+ anzuism_adj:0 "Anzuic"
+ anzuism_adherent:0 "Anzuist"
+ anzuism_adherent_plural:0 "Anzuists"
+ anzuism_desc:0 "A sect of $avian_group_adherent_plural$ who worship Anzu, the Raven God. As opposed to Rukhmar loving the sun and sky, Anzu prefer the cool of the shade and the peace of the twilight hours where he can be alone in quiet contemplation. He is ancient enemies with Sethe, who has cursed him long ago to be broken and flightless as well as many Anzu's arakkoa followers."
```
Comment (robmart) [inline localization/english/religion/wc_religion_l_english.yml:674]:
```suggestion
 anzuism_desc:0 "A sect of $avian_group_adherent_plural$ that worship Anzu, the Raven God. As opposed to Rukhmar who loves the sun and sky, Anzu prefers the cool of the shade and the peace of the twilight hours, where he can be solitary in peaceful contemplation. He is ancient enemies with Sethe, who has cursed him long ago to be broken and flightless as well as many of his arakkoa followers."
```

File: localization/english/religion/wc_religion_l_english.yml [hunk 2]
```diff
@@ -660,3 +660,27 @@
  masonry_adherent:0 "Mason"
  masonry_adherent_plural:0 "Masons"
  masonry_desc:0 "Arose among the guilds of masons, this branch of the Holy Light opposes itself to the Holy Light Church. They reject dogmas unpopular among the common folk such as the unshakable moral authority of the priests."
+ 
+ avian_group:0 "Avianism"
+ avian_group_adj:0 "Avian"
+ avian_group_adherent:0 "Avian"
+ avian_group_adherent_plural:0 "Avians"
+ avian_group_desc:0 "A group of faiths aroused among the arakkoa of Draenor consisting of four $avian_group_adherent$ gods: Rukhmar, Anzu, Sethe and the greatest king Terokk they worship as their fourth god."
```
Comment (KingofMushan) [inline localization/english/religion/wc_religion_l_english.yml:668]:
```suggestion
 avian_group_desc:0 "A group of faiths aroused among the arakkoa of Draenor consisting of four $avian_group_adherent$ gods: Rukhmar, Anzu, Sethe, and Terokk, the greatest of the Arakkoan kings now worshipped as a god."
```

File: localization/english/religion/wc_religion_l_english.yml [hunk 3]
```diff
@@ -660,3 +660,27 @@
...
+ anzuism_adj:0 "Anzuic"
+ anzuism_adherent:0 "Anzuist"
+ anzuism_adherent_plural:0 "Anzuists"
+ anzuism_desc:0 "A sect of $avian_group_adherent_plural$ who worship Anzu, the Raven God. As opposed to Rukhmar loving the sun and sky, Anzu prefer the cool of the shade and the peace of the twilight hours where he can be alone in quiet contemplation. He is ancient enemies with Sethe, who has cursed him long ago to be broken and flightless as well as many Anzu's arakkoa followers."
+ 
+ rukhmarism:0 "Rukhmarism"
+ rukhmarism_adj:0 "Rukhmaric"
+ rukhmarism_adherent:0 "Rukhmarist"
+ rukhmarism_adherent_plural:0 "Rukhmarists"
+ rukhmarism_desc:0 "A branch of $avian_group_adherent_plural$ who worship Rukhmar, the Sun Goddess. After another $avian_group_adherent$ god, Anzu, faded into darkness, she created the arakkoa to represent her grace and his cunning. As the mother of the arakkoa, Rukhmar said to be their main and traditional diety. Those who venerate her are believed to reunited with the mother in the skies."
```
Comment (robmart) [inline localization/english/religion/wc_religion_l_english.yml:680]:
```suggestion
 rukhmarism_desc:0 "A branch of $avian_group_adherent_plural$ who worship Rukhmar, the Sun Goddess. After the god Anzu faded into darkness, she created the arakkoa to represent her grace and his cunning. As the mother of the arakkoa, Rukhmar said to be their main and traditional diety. Those who venerate her are believed to reunite with their mother in the skies."
```

File: localization/english/religion/wc_religion_l_english.yml (hunk 1 above)
Comment (KingofMushan) [inline localization/english/religion/wc_religion_l_english.yml:674]:
```suggestion
 anzuism_desc:0 "A sect of $avian_group_adherent_plural$ who worship Anzu, the Raven God. Contrasting Rukhmar's love of the sun and sky, Anzu prefers the cool of the shade and the peace of the twilight hours when he can be alone in quiet contemplation. He is the ancient enemy of Sethe, who cursed him long ago to be broken and flightless, along with many of Anzu's arakkoa followers."
```

File: localization/english/religion/wc_religion_l_english.yml [hunk 4]
```diff
@@ -660,3 +660,27 @@
...
+
+ setheism:0 "Sethekk"
+ setheism_adj:0 "Sethekk"
+ setheism_adherent:0 "Sethekk"
+ setheism_adherent_plural:0 "Sethekk"
+ setheism_desc:0 "A sect of $avian_group_adherent_plural$ who worship Sethe, the Dead God. He was deceased by Anzu and Rukhmar, but before his death Sethe created the Curse. It caused some arakkoa to become flightless. Seen as an evil god by other $avian_group_adherent_plural$, $setheism_adherent_plural$ consider the Curse as a gift, a part of his divinity granted to his followers."
```
Comment (robmart) [inline localization/english/religion/wc_religion_l_english.yml:686]:
```suggestion
 setheism_desc:0 "A sect of $avian_group_adherent_plural$ who worship Sethe, the Dead God. He was slain by Anzu and Rukhmar, but before his death Sethe created a Curse, causing some arakkoa to become flightless. Seen as an evil god by other $avian_group_adherent_plural$, $setheism_adherent_plural$ consider the Curse as a gift, a part of his divinity granted to his followers."
```

**Review by robmart — CHANGES_REQUESTED** [review]
Some bits of the loc I think could be improved

File: localization/english/religion/wc_religion_l_english.yml (hunk 3 above)
Comment (KingofMushan) [inline localization/english/religion/wc_religion_l_english.yml:680]:
```suggestion
 rukhmarism_desc:0 "A branch of $avian_group_adherent_plural$ who worship Rukhmar, the Sun Goddess. After another $avian_group_adherent$ god, Anzu, faded into darkness, Rukhmar created the arakkoa to represent a fusion her grace and Anzu's cunning. As the mother of the arakkoa, Rukhmar is considered by most to be their main and most traditional diety. Those who venerate her are believed to reunite with the mother in the skies after their passing."
```

File: localization/english/religion/wc_religion_l_english.yml (hunk 4 above)
Comment (KingofMushan) [inline localization/english/religion/wc_religion_l_english.yml:686]:
```suggestion
 setheism_desc:0 "A sect of $avian_group_adherent_plural$ who worship Sethe, the Dead God. He was killed by Anzu and Rukhmar, but before his death, Sethe created the Curse, causing many arakkoa, as well as Anzu himself, to become flightless. Seen as an evil god by other $avian_group_adherent_plural$, $setheism_adherent_plural$ consider the Curse as a gift, a part of Sethe's divinity granted to his followers."
```

File: localization/english/religion/wc_religion_l_english.yml [hunk 5]
```diff
@@ -660,3 +660,27 @@
  masonry_adherent:0 "Mason"
  masonry_adherent_plural:0 "Masons"
  masonry_desc:0 "Arose among the guilds of masons, this branch of the Holy Light opposes itself to the Holy Light Church. They reject dogmas unpopular among the common folk such as the unshakable moral authority of the priests."
+ 
+ avian_group:0 "Avianism"
+ avian_group_adj:0 "Avian"
+ avian_group_adherent:0 "Avian"
+ avian_group_adherent_plural:0 "Avians"
+ avian_group_desc:0 "A group of faiths aroused among the arakkoa of Draenor consisting of four $avian_group_adherent$ gods: Rukhmar, Anzu, Sethe and the greatest King Terokk, who they worship as their fourth god."
```
Comment (robmart) [inline localization/english/religion/wc_religion_l_english.yml:668]:
```suggestion
 avian_group_desc:0 "A group of faiths aroused among the arakkoa of Draenor consisting of four $avian_group_adherent$ gods: Rukhmar, Anzu, Sethe and Terokk, the greatest of the Arakkoan kings now worshipped as a god."
```

File: localization/english/religion/wc_religion_l_english.yml [hunk 6]
```diff
@@ -660,3 +660,27 @@
...
+ 
+ anzuism:0 "Anzuism"
+ anzuism_adj:0 "Anzuic"
+ anzuism_adherent:0 "Anzuist"
+ anzuism_adherent_plural:0 "Anzuists"
+ anzuism_desc:0 "A sect of $avian_group_adherent_plural$ that worship Anzu, the Raven God. As opposed to Rukhmar who loves the sun and sky, Anzu prefers the cool of the shade and the peace of the twilight hours, where he can be solitary in peaceful contemplation. He is ancient enemies with Sethe, who has cursed him long ago to be broken and flightless as well as many of his arakkoa followers."
```
Comment (robmart) [inline localization/english/religion/wc_religion_l_english.yml]:
```suggestion
 anzuism_desc:0 "A sect of $avian_group_adherent_plural$ that worship Anzu, the Raven God. Contrasting Rukhmar's love of the sun and sky, Anzu prefers the cool of the shade and the peace of the twilight hours, when he can be solitary in peaceful contemplation. He is the ancient enemy of Sethe, who has cursed him long ago to be broken and flightless, along with many of Anzu's arakkoa followers."
```

File: localization/english/religion/wc_religion_l_english.yml (hunk 2 above)
Comment (zumbak04) [reply localization/english/religion/wc_religion_l_english.yml]: Merged with 14f2be06dd0f615d11951f673f6ea7aaa20b3647 in abfc725060e254b4959f510535ef1421e0f22dba.

File: localization/english/religion/wc_religion_l_english.yml (hunk 1 above)
Comment (zumbak04) [reply localization/english/religion/wc_religion_l_english.yml]: Merged with 14f2be06dd0f615d11951f673f6ea7aaa20b3647 in abfc725060e254b4959f510535ef1421e0f22dba.

File: localization/english/religion/wc_religion_l_english.yml (hunk 3 above)
Comment (zumbak04) [reply localization/english/religion/wc_religion_l_english.yml]: Merged with 14f2be06dd0f615d11951f673f6ea7aaa20b3647 in abfc725060e254b4959f510535ef1421e0f22dba.

File: localization/english/religion/wc_religion_l_english.yml (hunk 4 above)
Comment (zumbak04) [reply localization/english/religion/wc_religion_l_english.yml]: Merged with 14f2be06dd0f615d11951f673f6ea7aaa20b3647 in abfc725060e254b4959f510535ef1421e0f22dba.

File: localization/english/religion/wc_religion_l_english.yml [hunk 7]
```diff
@@ -660,3 +660,27 @@
...
+ 
+ rukhmarism:0 "Rukhmarism"
+ rukhmarism_adj:0 "Rukhmaric"
+ rukhmarism_adherent:0 "Rukhmarist"
+ rukhmarism_adherent_plural:0 "Rukhmarists"
+ rukhmarism_desc:0 "A branch of $avian_group_adherent_plural$ who worship Rukhmar, the Sun Goddess. After the god Anzu faded into darkness, she created the arakkoa to represent her grace and his cunning. As the mother of the arakkoa, Rukhmar said to be their main and traditional diety. Those who venerate her are believed to reunite with their mother in the skies."
```
Comment (robmart) [inline localization/english/religion/wc_religion_l_english.yml]:
```suggestion
 rukhmarism_desc:0 "A branch of $avian_group_adherent_plural$ who worship Rukhmar, the Sun Goddess. After the god Anzu faded into darkness, Rukhmar created the arakkoa to represent a fusion of her grace and Anzu's cunning. As the mother of the arakkoa, Rukhmaris considered by most to be their main and most traditional diety. Those who venerate her are believed be reunited with their mother in the skies after their passing."
```

File: localization/english/religion/wc_religion_l_english.yml [hunk 8]
```diff
@@ -660,3 +660,27 @@
...
+
+ setheism:0 "Sethekk"
+ setheism_adj:0 "Sethekk"
+ setheism_adherent:0 "Sethekk"
+ setheism_adherent_plural:0 "Sethekk"
+ setheism_desc:0 "A sect of $avian_group_adherent_plural$ who worship Sethe, the Dead God. He was slain by Anzu and Rukhmar, but before his death Sethe created a Curse, causing some arakkoa to become flightless. Seen as an evil god by other $avian_group_adherent_plural$, $setheism_adherent_plural$ consider the Curse as a gift, a part of his divinity granted to his followers."
```
Comment (robmart) [inline localization/english/religion/wc_religion_l_english.yml]:
```suggestion
 setheism_desc:0 "A sect of $avian_group_adherent_plural$ who worship Sethe, the Dead God. He was slain by Anzu and Rukhmar, but before his death, Sethe created a Curse, causing many arakkoa, as well as Anzu himself, to become flightless. Seen as an evil god by other $avian_group_adherent_plural$, $setheism_adherent_plural$ consider the Curse as a gift, a part of Sethe's divinity granted to his followers."
```

**Review by robmart — COMMENTED** [review]
Added Mushans suggestions on top of mine

File: localization/english/religion/wc_religion_l_english.yml (hunk 5 above)
Comment (zumbak04) [reply localization/english/religion/wc_religion_l_english.yml]: Done in c25c2cd14784e04ecf0138d1a7c5f7228595160b.

**Comment by robmart** [comment]
Oh, you already did that, whoops

**Review by robmart — APPROVED** [review]
Loc LGTM

**Review by KingofMushan — APPROVED** [review]
Looks good to me.

**Review by NovelConcept — COMMENTED** [review]
All in all, it seems good to me! There's just one thing I would note, regarding Ruhkmarism.

"While the ancient Apexis wielded the Light, the Adherents (of Rukhmar) used solar nature magic comparable to druids." I'm not sure how this works in the context of the mod, but if solar magic is going to be an aspect of Nature magic, I would have them prioritize Nature magic over Light magic. Possibly even over Arcane magic.

**Review by robmart — CHANGES_REQUESTED** [review]
Accoring to the description the Sethekk should be hated by the other Avian religions, however they are considered astray.
bild

**Comment by zumbak04** [comment]
> All in all, it seems good to me! There's just one thing I would note, regarding Ruhkmarism.
> 
> "While the ancient Apexis wielded the Light, the Adherents (of Rukhmar) used solar nature magic comparable to druids." I'm not sure how this works in the context of the mod, but if solar magic is going to be an aspect of Nature magic, I would have them prioritize Nature magic over Light magic. Possibly even over Arcane magic.

I consider it a very strange categorization by Blizzard that should be ignored for the sake of our categorization.
It raises the question of what is druid solar magic then? Why some draw Light magic from Sun (like the Sunwalkers and high elves) and others draw some special kind of Nature magic.

**Comment by zumbak04** [comment]
> Accoring to the description the Sethekk should be hated by the other Avian religions, however they are considered astray.
> bild

Changed in 035e4859ea7b4c5065199898cbba53ad4d82cfab.

**Comment by NovelConcept** [comment]
> > All in all, it seems good to me! There's just one thing I would note, regarding Ruhkmarism.
> > "While the ancient Apexis wielded the Light, the Adherents (of Rukhmar) used solar nature magic comparable to druids." I'm not sure how this works in the context of the mod, but if solar magic is going to be an aspect of Nature magic, I would have them prioritize Nature magic over Light magic. Possibly even over Arcane magic.
> 
> I consider it a very strange categorization by Blizzard that should be ignored for the sake of our categorization.
> It raises the question of what is druid solar magic then? Why some draw Light magic from Sun (like the Sunwalkers and high elves) and others draw some special kind of Nature magic.

Well _technically,_ the Sunwalkers actually use this "solar nature magic" as well, and them being Paladins in-game is purely mechanical. Lore-wise, they're basically solar druids who wear armor. That being said, it does kind of give an argument for Light being close enough to Solar Nature for our purposes, so it's all good.

**Review by NovelConcept — APPROVED** [review]

**Review by robmart — APPROVED** [review]
LGTM

**Pull request merged**